src/Fieldset/README.md

```diff
@@ -0,0 +1,29 @@
+# Fieldset
+
+Wraps many inputs in a border and allows validating them as one form control.
+
+## API
+
+### `dangerIconDescription` (string or element; optional)
+`dangerIconDescription` can be used to provide a screen-reader description of the "exclamation" icon used in the validation message displayed when `isValid` is false and `themes` includes "danger". The default is an empty string.
+
+### `children` (string, element, or arrays of strings or elements; optional)
+`children` specifies the elements that the Fieldset will wrap. The default is an empty string.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the input component. The default is an empty array.
+
+### `id` (string; optional)
+`id` specifies a unique identifier for the fieldset.  It will be used to create an id for the ValidationMessage that is referenced by an `aria-describedby` on the `fieldset`. The default is an auto-generated id of the form `fieldsetN` where N is an auto-incremented integer.
+
+### `isValid` (boolean; optional)
+`isValid` specifies whether the whole fieldset has validated correctly. Consider updating this from an `onSubmit` handler of a parent `form` element. The default is true.
+
+### `legend` (string or element; required)
+`legend` specifies the label to be used for the overarching fieldset. This can be a string or element type. It appears at the top of the fieldset, inline with the border.
+
+### `themes` (array of strings; optional)
+`themes` specifies the themes to apply to the input (e.g. "danger"). The default is an empty array.
```

Comment:
Maybe clarify these are one of [the eight `Bootstrap` contextual variations](https://getbootstrap.com/docs/4.0/components/badge/#contextual-variations)?

Reply:
Isn't that docs page specific to badges though? I don't see any docs pages in Bootstrap about the different conceptual themes they have (danger, warning, info).

Reply:
https://v4-alpha.getbootstrap.com/utilities/colors/

Reply:
Thanks Farhannah, I found the v4.0 docs for that and linked it in the docs here: https://getbootstrap.com/docs/4.0/utilities/colors/